docs/source/federation-version-support.mdx

```diff
@@ -23,7 +23,15 @@ The table below shows which version of federation each router release is compile
     <tbody>
         <tr>
         <td>
-            v1.0.0-alpha.3 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
+            v1.0.0-rc.0 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
+        </td>
+        <td>
+            2.1.2-alpha.1
+        </td>
+    </tr>
+    <tr>
+        <td>
+            v1.0.0-alpha.3
```

Comment:
I don't understand the order here 👀

Reply:
@o0Ignition0o Do you find it any easier to grok in [the rendered version](https://deploy-preview-1775--apollo-router-docs.netlify.app/federation-version-support)?

Reply:
ooooohhh classic! gotchu thanks!